docs/source/visualization/interval_color_mapping.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>No text describing what this does, the list comprehension is kind of dense code given no explanation.</p><p><code>zip</code> is a bit of a magic function, might want to say what that line does. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>